ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/infinispan/protostream/main/module.xml

```diff
@@ -32,8 +33,9 @@
     </resources>
 
     <dependencies>
-        <module name="javax.api"/>
+        <!--WFLY-14219 Remove deprecated <module name="javax.api"/> -->
```

Comment:
This needs a dep on java.logging added. I tried that at #13840 and the many failures went away.

Reply:
Thanks, resolved.